packages/database/src/getConnectionConfig.js

```diff
@@ -5,18 +5,18 @@
 
 import { requireEnv, getEnvVarOrDefault } from '@tupaia/utils';
 
-const getServerConfig = () => ({
+const getServerConfig = ({ userEnv = 'DB_USER', passEnv = 'DB_PASSWORD' }) => ({
```

Comment:
Oooh love this change, very clean and would be nicely extensible!

Reply:
^The real @rohan-bes agrees with everything this imposter says 😂